ui/desktop/app/styles/app.scss

```diff
@@ -29,13 +29,34 @@
 
   .rose-layout-global-body {
     max-height: 100vh;
-    overflow: hidden;
+    overflow: auto;
+
+    main {
+      flex-grow: 1;
+      overflow: auto;
+    }
   }
 
   .rose-layout-sidebar-sidebar,
   .rose-layout-sidebar-body {
     overflow-y: scroll;
   }
+
+  // Windows OS specific overflow.
```

Comment:
How do we apply these CSS JUST to windows OS? or we applied for both clients but just affects windows?

Reply:
The style is applied selectively based on platform as shown [here](https://github.com/hashicorp/boundary-ui/pull/588/files#diff-de7e5aba3ecc486ff8ba63bcd57bf8e6660eb3d3dd45530d712a2417548bf65cR4). I cleaned up.

Reply:
Ahh makes all the sense, I miss the condition on the view. :thumbsup: